.nodeset.yml

```diff
@@ -9,23 +9,3 @@ sets:
     nodes:
```

Comment:
What is this file used for? Can it go entirely?

Reply:
This is just some old left over config that can be removed.